Makefile

```diff
@@ -76,7 +76,7 @@ all-push-skip-if-present:
 
 .PHONY: build_image
 build_image:
-	DOCKER_BUILDKIT=1 docker buildx build -f ${DOCKERFILE} -t=${IMAGE}:${TAG} --platform=${PLATFORM} .
+	DOCKER_BUILDKIT=1 docker buildx build -f ${DOCKERFILE} -t=${IMAGE}:${TAG} --platform=${PLATFORM} --build-arg VERSION=$(VERSION) .
```

Comment:
Looks like we already embed CSI Driver's version in Go binary, see this line in `Makefile`: https://github.com/awslabs/mountpoint-s3-csi-driver/blob/main/Makefile#L23